prov/efa/src/efa_hmem.c

```diff
@@ -148,6 +148,8 @@ static int efa_domain_hmem_info_init_cuda(struct efa_domain *efa_domain)
 	int ibv_access = IBV_ACCESS_LOCAL_WRITE | IBV_ACCESS_REMOTE_READ;
 	size_t len = ofi_get_page_size() * 2, tmp_value;
 	int ret;
+	int dmabuf_fd;
```

Comment:
nit Suggest initializing these variables.

Reply:
Don't think so. `fd` and `offset` are only used when `cuda_get_dmabuf_fd` returns success, where they got assigned. We don't need to initialize them.